src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/tests/latest/test_ams_content_key_policy_scenarios.py

```diff
@@ -71,3 +70,81 @@ def test_content_key_policy_show_basic(self, storage_account_for_show):
             self.check('options[0].configuration.odatatype', '{configurationODataType}'),
             self.check('options[0].restriction.odatatype', '{restrictionODataType}')
         ])
+
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer(parameter_name='storage_account_for_create')
+    def test_content_key_policy_delete(self, storage_account_for_create):
```

Comment:
event is for delete yet storage account is for create.. silly thing but we need to make it consistent